src/content/tutorials/manage-projects-with-lerna/_index.md

```diff
@@ -66,16 +68,22 @@ const handler = request => {
 
 After adding an identical `dependency` to `private-api/package.json`, you can run `lerna bootstrap` again, and begin sharing code between your projects.
 
-When you're ready to deploy your codebases, you can coordinate deploying them simultaneously by defining scripts in `package.json` that can be read by `lerna run`:
+When you're ready to deploy your codebases, you can coordinate deploying them simultaneously by defining scripts in each package's `package.json` file, that can be read by `lerna run`:
+
+In `handlers/public-api/package.json`:
 
-In `handlers/package.json`:
 ```json
 {
   "name": "public-api",
   "scripts": {
     "publish": "wrangler publish"
   }
 }
+```
+
+In `handlers/private-api/package.json`:
```

Comment:
The example below doesn't have a handlers directory, I think it's just called packages? Not sure if both are ok but would be good for example to match.